common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
She's not, she ended my scheme by discovering I lead it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117039098-709c2100-ad19-11eb-9e80-cf56c0979a4c.png)

</details>

Reply:
Turns out i inverted the stasis check so everything that ended the scheme gave this message, should be fixed now